rewrite-maven/src/main/java/org/openrewrite/maven/tree/Pom.java

```diff
@@ -115,6 +115,9 @@ public static int getModelVersion() {
     @Builder.Default
     List<Plugin> pluginManagement = emptyList();
 
+    @Builder.Default
+    List<String> modules = emptyList();
```

Comment:
To avoid confusion & future changes, perhaps it helps to already adopt the Maven 4 convention of [calling these subprojects](https://github.com/openrewrite/rewrite/discussions/4553#discussioncomment-10996072)?

Reply:
Sounds good, that takes away the ambiguity!

Reply:
I've taken both `modules` and `subProjects` into account for the `RawPom` and named it `subProjects` anywhere beyond there